src/main/java/com/example/bookstoreproject/persistence/user/UserEntity.java

```diff
@@ -0,0 +1,35 @@
+package com.example.bookstoreproject.persistence.user;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.util.UUID;
+
+@Table(name = "users")
+@Getter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class UserEntity {
+
+    @Id
+    private UUID id;
+    private String username;
+    private String password;
+    @Column(name = "firstname")
```

Comment:
```suggestion
    @Column(name = "first_name")
```

Update migration script, do the same with lastname

Reply:
Có thể bỏ hẳn annontation luôn sau khi rename

Reply:
DONE